include/crypto/evp.h

```diff
@@ -250,7 +249,7 @@ struct evp_md_st {
     int (*md_ctrl) (EVP_MD_CTX *ctx, int cmd, int p1, void *p2);
 
     /* New structure members */
-    /* TODO(3.0): Remove above comment when legacy has gone */
+    /* Above comment to be removed when #legacy has gone */
```

Comment:
Why #legacy here but nowhere else?  I'd suggest removing the #

Reply:
There are other places as well. I started with that so it would be easy to grep when the time coms

Reply:
I agree, though, that it should be consistent. Contrary to @paulidale, I suggest adding the # in the other spots

Reply:
It looks like 9 #legacy and 8 legacy.  I'm happy either way.

Reply:
There are actually zillion places where legacy without # is used.

OK, when I look for legacy in a comment then I can still find about 340 occurrences. I'll change this to legacy without #. If anyone wants to do a global replacement with #legacy then this should be a different PR. But I do not think it is worth it.